docs/eventing/samples/sequence/sequence-reply-to-event-display/README.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "Sequence Wired to event-display"
```

Comment:
Nice write up, @vaikas-google ! I'll review this later today.

Reply:
This PR has a lot more than I realized! Here's my first round of edits; still have a little more I'd like to do, but wanted to get these over to you.